src/components/Visualizations/inflictorWithValue.jsx

```diff
@@ -39,7 +40,9 @@ const tooltipContainer = thing => (
       <div
         dangerouslySetInnerHTML={{
           __html: thing.cmb
-            .replace('/apps/dota2/images/tooltips/mana.png', `${API_HOST}/apps/dota2/images/tooltips/mana.png`)
+            .replace('src="/apps/dota2/images/tooltips/mana.png"', `src="${API_HOST}/apps/dota2/images/tooltips/mana.png"`)
```

Comment:
Hmm, I guess we shouldn't put API_HOST in dotaconstants, but perhaps we should be stripping out cdn.dota2.com?